test/server.cpp

```diff
@@ -1052,17 +1061,6 @@ TEST_F(ServerTest, Http400HtmlError)
     <p>
       Too many books requested (4) where limit is 3
     </p>
-)"  },
-    { /* url */ "/ROOT%23%3F/search?content=zimfile",
-      expected_kiwix_response_data==R"({ "CSS_URL" : false, "PAGE_HEADING" : { "msgid" : "400-page-heading", "params" : { } }, "PAGE_TITLE" : { "msgid" : "400-page-title", "params" : { } }, "details" : [ { "p" : { "msgid" : "invalid-request", "params" : { "url" : "/ROOT%23%3F/search?content=zimfile" } } }, { "p" : { "msgid" : "no-query", "params" : { } } } ] })" &&
-      expected_body==R"(
-    <h1>Invalid request</h1>
-    <p>
-      The requested URL "/ROOT%23%3F/search?content=zimfile" is not a valid request.
-    </p>
-    <p>
-      No query provided.
-    </p>
```

Comment:
Why have this and other test points been removed in this commit?

These changes should be moved to the last commit (the update in the backend)!